README.md

```diff
@@ -46,12 +51,31 @@ Note that this sequence also works for branch based pull requests, as you'd expe
 ## Authentication
 
 Choose one of these authentication methods for the `repository_dispatch` helper action, `peter-evans/slash-command-dispatch`, in [`ok-to-test.yml`](.github/workflows/ok-to-test.yml):
+
 - [Personal access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) with `repo` scope
 - [OAuth "app" token](https://developer.github.com/v3/#oauth2-token-sent-in-a-header) with `repo` scope
-- ⭐️ Preferred: [GitHub App installation access token](https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#authenticating-as-an-installation) with `contents: write` and `metadata: read` permissions
+- ⭐️ Preferred: [GitHub App installation access token](https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#authenticating-as-an-installation) with `contents: write` and `metadata: read` permissions. See [Creating A GitHub App](#creating-a-github-app) for a basic overview of how to do this.
 
 GitHub Apps have distinct identities on GitHub – no seat taken up by a machine account, no potential for leaking your personal credentials, and no rate limit sharing!
 
+### Creating a GitHub App
+
+Here we are using a GitHub App as an authentication entity, rather than a traditional coding application. Below are some brief instructions on how to setup a GitHub App for this purpose, note that there are other methods of creating a GitHub App such as with a manifest file. _(The instructions below are for setting up an app within your user, but you can also do it for your organisation.)_
```

Comment:
```suggestion
Here we are using a GitHub App as an authentication entity. Below are some brief instructions on how to setup a GitHub App for this purpose, note that there are other methods of creating a GitHub App such as with a manifest file (e.g. one similar to [`app.yml`](/app.yml)). _(The instructions below are for setting up an app within your user, but you can also do it for your organization.)_
```

Reply:
@imjohnbo - rather than remove, I've re-worked the 'traditional coding application' bit. This is something that took me a little bit of time to realise, so I thought it important to call out.

Have implemented the rest of your changes. I'll leave the organisation 🇬🇧 vs organization 🇺🇸 argument for another time... as I'm likely to be outnumbered here! 😂